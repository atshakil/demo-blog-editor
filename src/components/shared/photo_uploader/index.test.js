import React from 'react';
import { library } from '@fortawesome/fontawesome-svg-core';
import { fas } from '@fortawesome/free-solid-svg-icons';
library.add(fas);

import { testWithIntl } from '../../../services/test';
import PhotoUploader from '.';

it('renders without crashing', () => {
  const component = testWithIntl(<PhotoUploader />);
  let tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
