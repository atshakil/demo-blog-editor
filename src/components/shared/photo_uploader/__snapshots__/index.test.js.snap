// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="photo-uploader row"
>
  <div
    aria-disabled={false}
    className="dropzone w-100"
    onClick={[Function]}
    onDragEnter={[Function]}
    onDragLeave={[Function]}
    onDragOver={[Function]}
    onDragStart={[Function]}
    onDrop={[Function]}
    style={
      Object {
        "position": "relative",
      }
    }
  >
    <div
      className="dropzone-hint col"
    >
      <div
        className="hint hint-icon row"
      >
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-camera-retro fa-w-16 fa-4x "
          data-icon="camera-retro"
          data-prefix="fas"
          role="img"
          style={Object {}}
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M48 32C21.5 32 0 53.5 0 80v352c0 26.5 21.5 48 48 48h416c26.5 0 48-21.5 48-48V80c0-26.5-21.5-48-48-48H48zm0 32h106c3.3 0 6 2.7 6 6v20c0 3.3-2.7 6-6 6H38c-3.3 0-6-2.7-6-6V80c0-8.8 7.2-16 16-16zm426 96H38c-3.3 0-6-2.7-6-6v-36c0-3.3 2.7-6 6-6h138l30.2-45.3c1.1-1.7 3-2.7 5-2.7H464c8.8 0 16 7.2 16 16v74c0 3.3-2.7 6-6 6zM256 424c-66.2 0-120-53.8-120-120s53.8-120 120-120 120 53.8 120 120-53.8 120-120 120zm0-208c-48.5 0-88 39.5-88 88s39.5 88 88 88 88-39.5 88-88-39.5-88-88-88zm-48 104c-8.8 0-16-7.2-16-16 0-35.3 28.7-64 64-64 8.8 0 16 7.2 16 16s-7.2 16-16 16c-17.6 0-32 14.4-32 32 0 8.8-7.2 16-16 16z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
      </div>
      <div
        className="hint row"
      >
        Úp hình thành phẩm
      </div>
    </div>
    <input
      accept="image/*"
      autoComplete="off"
      disabled={false}
      multiple={false}
      onChange={[Function]}
      style={
        Object {
          "bottom": 0,
          "left": 0,
          "opacity": 0.00001,
          "pointerEvents": "none",
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
      type="file"
    />
  </div>
</div>
`;
